hs_core/hydroshare/resource.py

```diff
@@ -1352,6 +1319,8 @@ def publish_resource(user, pk):
 
         resource.raccess.save()
         resource.save()
+        from hs_core.tasks import notify_developers_of_publication_failure
+        notify_developers_of_publication_failure.apply_async((pk, str(e)))
```

Comment:
You should set the `DataciteSubmissionStatus.FAILURE` enum on the doi in this case